flink-runtime/src/main/java/org/apache/flink/runtime/blob/BlobService.java

```diff
@@ -18,61 +18,31 @@
 
 package org.apache.flink.runtime.blob;
 
-import org.apache.flink.api.common.JobID;
-
 import java.io.Closeable;
-import java.io.File;
-import java.io.IOException;
 
 /**
  * A simple store and retrieve binary large objects (BLOBs).
  */
 public interface BlobService extends Closeable {
 
 	/**
-	 * Returns the path to a local copy of the (job-unrelated) file associated with the provided
-	 * blob key.
-	 *
-	 * @param key blob key associated with the requested file
-	 * @return The path to the file.
-	 * @throws java.io.FileNotFoundException when the path does not exist;
-	 * @throws IOException if any other error occurs when retrieving the file
-	 */
-	File getFile(BlobKey key) throws IOException;
-
-	/**
-	 * Returns the path to a local copy of the file associated with the provided job ID and blob key.
+	 * Returns a BLOB service for accessing permanent BLOBs.
 	 *
-	 * @param jobId ID of the job this blob belongs to
-	 * @param key blob key associated with the requested file
-	 * @return The path to the file.
-	 * @throws java.io.FileNotFoundException when the path does not exist;
-	 * @throws IOException if any other error occurs when retrieving the file
+	 * @return BLOB service
 	 */
-	File getFile(JobID jobId, BlobKey key) throws IOException;
+	PermanentBlobService getPermanentBlobStore();
```

Comment:
I'm not so sure about the naming here. The `BlobStore` is actually something else than the `BlobService`. Would be good if we use a consistent naming for the different things (this could also include renaming some of the entities).